falkordb-node/node-entrypoint.sh

```
@@ -106,29 +106,26 @@ fi
 
 if [[ ! -s "$FALKORDB_HOME/run_bgrewriteaof" && ! -f "$FALKORDB_HOME/run_bgrewriteaof" ]]; then
   echo "Creating run_bgrewriteaof script"
-  echo """#!/bin/bash
+  echo "
+      #!/bin/bash
       set -e
+      AOF_FILE_SIZE_TO_MONITOR=\${AOF_FILE_SIZE_TO_MONITOR:-5}
+      ROOT_CA_PATH=\${ROOT_CA_PATH:-/etc/ssl/certs/GlobalSign_Root_CA.pem}
+      TLS_CONNECTION_STRING=$(if [[ \$TLS == "true" ]]; then echo "--tls --cacert \$ROOT_CA_PATH"; else echo ""; fi)
```

Shellcheck (0.10.0)

[warning] This expression is constant. Did you forget the $ on a variable?

(SC2050)